metriq_gym/schemas/qml_kernel.schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$id": "metriq-gym/qml_kernel.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",  
+  "title": "QML Kernel",
+  "type": "object",
+  "properties": {
+    "benchmark_name": {
+      "type": "string",
+      "const": "QML Kernel"
+    },
+    "num_qubits": {
+      "type": "integer",
+      "minimum": 2,
+      "examples": [10]
+    },
+    "shots": {
+      "type": "integer",
+      "default": 1000,
+      "minimum": 1,
+      "examples": [1000]
+    },
+    "trials": {
```

Comment:
If trials are indeed unused, be sure to remove them here as well.

Reply:
Good idea, I will remove it for now, and introduce it in a follow-up PR.